package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "madmarkdown-parser",
+  "dependencies": {
+    "maddoc": "file:madlib_modules/https___github_com_madlib_lang_maddoc_archive_refs_heads_master_zip",
```

Comment:
I don’t remember why but this is wrong and does weird things with npm. There’s specifically a save and restore mechanism in the package installer for package.json to avoid this.